src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Splitter/BitSplitter.scss

```
@@ -3,9 +3,105 @@
+ * to zero, which is what pins it while the other one takes up the rest.
+ *
+ * The basis is the same property on both axes, so one pair of variables carries the width of a splitter
+ * laid out in a row and the height of one laid out in a column, and a size given as a percentage keeps
+ * its proportions while the container is resized.
+ */
+.bit-spl-pnl {
+    min-width: 0;
+    min-height: 0;
+    overflow: auto;
+    flex-shrink: 1;
+    // The flex basis of a panel is the size the drag measured off the element, and the two of them are only
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)